cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -30,23 +27,19 @@
                     <% } %>
                 <% } %>
             <% } %>
-        </ul>
-    </div>
-    <div class="tab" id="tab2">
-        <ul class="new-component-template">
             <% for (var i = 0; i < templates.length; i++) { %>
                 <% if (templates[i].tab == "hint") { %>
```

Comment:
Where in Python is the `tab` information specified? I am wondering if we can send the templates in the right order from Python rather than doing two loops here.